test/unit/contextResponses/contextResponseNumericWithFixedTimeInstantUpdate.json

```diff
@@ -0,0 +1,29 @@
+{
+  "contextResponses": [
+    {
+      "contextElement": {
+        "attributes": [
+          {
+            "name": "attrNameNumeric",
+            "type": "attrType",
+            "value": "222",
+            "metadatas": [
+              {
+                "name": "TimeInstant",
+                "type": "ISO8601",
+                "value": "1970-01-01T00:00:00.000"
```

Comment:
Good catch! In fact, the issue was the lack of the "Z" at the end of the dates... I mean, it should be ""1970-01-01T00:00:00.000Z" for them to be UTC dates... ;) Thanks!